### PR TITLE
Feature/60762 update radio button text on review details page

### PR DESCRIPTION
**Pull request by jenniemc**
## Summary
This PR updates on the confirmation and review details page. When the user selects the reason for the appointment, the description of the selection should match the selection on the "Choose your reason for the appointment" page.

## Acceptance criteria

- [x] Given the user is in the direct schedule workflow 
       When the user selects `This is a routine or follow up visit` and enter the comments 
       Then the review details and confirmation page will display `Routine/Follow-up`

- [x] Given the user is in the direct schedule workflow 
        When the user selects `I have a medical issue` and enter the comments
        Then the review details and confirmation page will display `New medical issue`

- [x] Given the user is in the direct schedule workflow 
        When the user selects ` I have a concern or question about my medication` and enter the comments
        Then the review details and confirmation details page will display `Medication concern`

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#60762

## Testing done

unit test

## Screenshots

Routine/Follow-up
Screenshot 2023-10-05 at 1 50 35 PM
---
Screenshot 2023-10-05 at 1 50 53 PM

___
New medical issue
Screenshot 2023-10-05 at 1 52 10 PM
---
Screenshot 2023-10-05 at 1 52 33 PM

___
Medication concern
Screenshot 2023-10-05 at 1 53 56 PM
---
Screenshot 2023-10-05 at 1 54 13 PM

## What areas of the site does it impact?

VAOS review and confirmation page

### CODE REVIEW

File: src/applications/vaos/new-appointment/redux/helpers/getReasonCode.js [hunk 1]
```diff
@@ -16,9 +16,10 @@ export function getReasonCode({ data, isCC, isAcheron, isDS }) {
   const apptReasonCode = PURPOSE_TEXT_V2.find(
     purpose => purpose.id === data.reasonForAppointment,
   )?.commentShort;
+  // code is utilized when Acheron is off; need to cleanup
   const code = PURPOSE_TEXT_V2.filter(purpose => purpose.id !== 'other').find(
```
Comment (young1205) [inline src/applications/vaos/new-appointment/redux/helpers/getReasonCode.js:20]: Did this comment get removed intentionally?

File: src/applications/vaos/new-appointment/redux/helpers/getReasonCode.js [hunk 2]
```diff
@@ -61,6 +62,7 @@ export function getReasonCode({ data, isCC, isAcheron, isDS }) {
         : null,
   };
   if (isAcheron) return reasonCodeBody;
+  // the below return is utilized when Acheron is off - need to cleanup
   return {
```
Comment (young1205) [inline src/applications/vaos/new-appointment/redux/helpers/getReasonCode.js:66]: Is this comment removal intended as well ?

Comment (jenniemc) [reply src/applications/vaos/new-appointment/redux/helpers/getReasonCode.js:66]: The comment is added as reminder when we do cleanup for Acheron toggle flag removal.

File: src/applications/vaos/new-appointment/redux/helpers/getReasonCode.js (hunk 1 above)
Comment (jenniemc) [reply src/applications/vaos/new-appointment/redux/helpers/getReasonCode.js:20]: This comment is added as reminder when we do cleanup for Acheron toggle flag removal.

**Comment by jenniemc** [comment]
@young1205, Would you review and approve this PR?

**Review by young1205 — APPROVED** [review]
The second workflow should read `I have a new medical issue` but everything aside from that looks correct and that is not really in the scope of this ticket

**Review by vbahinwillit — APPROVED** [review]
LGTM!

**Review by ataker — APPROVED** [review]

**Pull request merged**